README.md

```diff
@@ -13,6 +13,7 @@ Config and instructions for a DiSARM OpenFaas cluster from scratch.
 1. Create network `traefik-net` with overlay driver: `docker network create -d overlay --attachable traefik-net`
 1. Ensure `docker-compose.yml`, `openfaas-docker-compose.yml` and `traefik.toml` contain references to the correct domain/subdomains.
 1. Start the stack: `docker stack deploy -c docker-compose.yml rp`
+1. Add _secret_ called `ssl-cert`, to allow Squid's SSL certificate to be accesssed by functions: `docker secret create ssl-cert ./squid/cert/private.pem`.
```

Comment:
Once we've tested `fn-cov-extr` with this cache, let's add a section to the README that explains how we can use the cache for other functions.